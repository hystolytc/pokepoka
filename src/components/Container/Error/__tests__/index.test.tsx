import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import { Error } from '../index'

const describeComponent = '<Error /> test case'

describe(describeComponent, () => {
  const { container } = render(<Error />)

  test(`Render ${describeComponent} correctly`, () => {
    expect(container).toMatchSnapshot()
  })
})