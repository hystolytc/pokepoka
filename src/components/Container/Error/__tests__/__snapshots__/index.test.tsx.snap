// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> test case Render <Error /> test case correctly 1`] = `
<div>
  <div
    class="css-zn5vyk"
  >
    <img
      alt="pokemon ball"
      class="css-mh31mg"
      height="100"
      src="pokeball-broken.png"
      width="100"
    />
    <p
      class="css-1b4xw04"
    >
      Woops, something went wrong!
    </p>
  </div>
</div>
`;
